ui/design-system-core/src/components/AppAvatar/index.tsx

```diff
@@ -51,15 +51,15 @@ const AppAvatar: React.FC<IAppAvatarProps> = props => {
     }.webp`;
   }
 
-  const className = `box-border cursor-${
+  const className = apply`box-border cursor-${
```

Comment:
extract this into multiple vars to avoid the multiple conditionals in the same style, and for conditional classes use the [cx utility ](https://twind.style/reference#utilities)